CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 Change Log
 ==========
 
+## Version 1.0.0-alpha.10
+
+_2021-03-04_
```

Comment:
Didn't we also upgrade to kotlin 1.4.30 in this release? That should be in here too.

Reply:
Looking through the changelog, I'm also seeing:
- Fixes core error message in ViewRegistry.buildView (#286)
- Make ModalContainer look for the parent lifecycle using ViewTreeLifecycleOwner. (#315)
- Stop leaking the viewbinding dependency to consumers. (#320)
- Introduce support for ViewTreeSavedStateRegistryOwner in Android container views. (#284) 

 That last one should probably be right beside the lifecycle owner one.

Reply:
Thanks. It was tricky distinguishing what was new and what came in from the compose import.

Reply:
https://github.com/square/workflow-kotlin/pull/349

ViewTreeSavedStateRegistryOwner was already in there.